src/main/resources/com/google/api/codegen/java/method_sample.snip

```diff
@@ -80,9 +85,15 @@
 
 @snippet asyncOperationCallableMethodSampleCode(apiMethod, initCode)
   {@initCode(initCode)}
-  OperationFuture<{@apiMethod.callableMethod.genericAwareResponseType}> future = {@sampleFutureMethodCall(apiMethod)};
```

Comment:
Do you know why we used `genericAwareResponseType` in the first place?

Reply:
It seems it's converting `Empty` to `Void`. For exmaple, the callable methods for those rpcs who return `Empty` would return `ApiFuture<Void>` instead to be more Java idiomatic. But it seems we do not do this conversion for `LongRunningCallable`s.